src/Cake.Unity/SeekersOfEditors/SeekerOfEditors.cs

```diff
@@ -52,7 +50,10 @@ public IReadOnlyCollection<UnityEditorDescriptor> Seek()
             return editors.ToList();
         }
 
-        protected abstract string SearchPattern { get; }
+        private List<FilePath> GetCandidates(string[] searchPatterns) =>
```

Comment:
Please put `private` method below `protected` methods. As they belong to class contract while `private` does not.